packages/grid/data-grid/package.json

```
@@ -31,6 +31,7 @@
   },
   "dependencies": {
     "@material-ui/utils": "^5.0.0-alpha.14",
+    "clsx": "^1.1.1",
```

A small detail, 

```suggestion
    "clsx": "^1.0.4",
```

x2 might be better https://github.com/mui-org/material-ui/blob/8cb115b4fa8bb9ba0690b50bae1c1ffbbb7346d1/packages/material-ui/package.json#L72

---

It's not really important